examples/misc/bin/try_dart/functions.dart

```diff
@@ -0,0 +1,22 @@
+// A function declaration.
+int timesTwo(int x) {
+  return x * 2;
+}
+
+// Arrow syntax is shorthand for `{ return expr; }`.
+int timesFour(int x) => timesTwo(timesTwo(x));
+
+// Functions are objects.
+int runTwice(int x, int Function(int) f) {
+  for (var i = 0; i < 2; i++) {
+    x = f(x);
```

Comment:
**Difference relative to the original source:** The analyzer wouldn't allow an untyped `f` to be used to assigned to the typed `x`, so I declared `f` as a function from int to int.

Reply:
Code change undone.